core/server/apps/subscribers/lib/views/subscribe.hbs

```diff
@@ -25,7 +25,7 @@
             <div class="gh-flow">
                 <header class="gh-flow-head">
                     <nav class="gh-flow-nav">
-                        <a href="{{@blog.url}}" class="gh-flow-back"><i class="icon-arrow-left"></i> Back</a>
+                        <a href="{{@blog.url}}" class="gh-flow-back" onclick="window.history.go(-1); return false;"><i class="icon-arrow-left"></i> Back</a>
```

Comment:
Apparently `{{subscribed_url}}` is always empty for me. Is there some doku? Couldn't find anything here: https://themes.ghost.org/v0.8.0/docs/subscribers